packages/angular/src/app/pages/common-profile/common-profile.component.html

```diff
@@ -0,0 +1,109 @@
+<dx-scroll-view class="view-wrapper-scroll">
+  <div class="view-wrapper">
+    <dx-toolbar>
+      <dxi-item location="before">
+        <div class="header-text">User Profile</div>
+      </dxi-item>
+      <dxi-item location="after" locateInMenu="never">
+        <dx-button
+          text="Cancel"
+          stylingMode="contained"
+        ></dx-button>
+      </dxi-item>
+      <dxi-item location="after" locateInMenu="never">
+        <dx-button
+          [disabled]="!isDataChanged"
+          text="Save"
+          icon="save"
+          type="default"
+          stylingMode="contained"
+          (click)="save()"
+        ></dx-button>
+      </dxi-item>
+    </dx-toolbar>
+
+    <dx-load-panel
+      [visible]="isLoading"
+      [showPane]="false"
+      container=".view-wrapper-scroll"
+      [position]="{ of: 'common-profile' }"
+    ></dx-load-panel>
+    <div *ngIf="!isLoading" class="cards-container">
```

Comment:
Let's change formatting here:

```suggestion
    <div *ngIf="!isLoading" 
         class="cards-container">
```